CONTRIBUTING.rst

```diff
@@ -144,17 +143,15 @@ Running System Tests
 
    # Run all system tests
    $ nox -s system-3.8
-   $ nox -s system-2.7
 
    # Run a single system test
    $ nox -s system-3.8 -- -k <name of test>
 
 
   .. note::
 
-      System tests are only configured to run under Python 2.7 and
-      Python 3.8. For expediency, we do not run them in older versions
-      of Python 3.
+      System tests are only configured to run under Python 3.8.
```

Comment:
WE run system tests under 3.9 **and** 3.9.

Reply:
Good catch!

Reply:
Fixed via googleapis/synthtool#1132. I've updated the SHA of the post processor image in `.github/.OwlBot.lock.yaml` to pull in these changes.